scripts/plugins/storage/postgres/init.sql

```diff
@@ -984,6 +984,8 @@ DELETE FROM fledge.users;
 INSERT INTO fledge.users ( uname, real_name, pwd, role_id, description )
      VALUES ('admin', 'Admin user', '495f7f5b17c534dbeabab3da2287a934b32ed6876568563b04c312be49e8773299243abd3881d13112ccfb67c4fb3ec8231406474810e1f6eb347d61c63785d4:672169c60df24b76b6b94e78cad800f8', 1, 'admin user'),
             ('user', 'Normal user', '495f7f5b17c534dbeabab3da2287a934b32ed6876568563b04c312be49e8773299243abd3881d13112ccfb67c4fb3ec8231406474810e1f6eb347d61c63785d4:672169c60df24b76b6b94e78cad800f8', 2, 'normal user');
+INSERT INTO fledge.users ( uname, real_name, pwd, role_id, description, access_method)
+     VALUES ('systemctl', 'Systemctl user', '495f7f5b17c534dbeabab3da2287a934b32ed6876568563b04c312be49e8773299243abd3881d13112ccfb67c4fb3ec8231406474810e1f6eb347d61c63785d4:672169c60df24b76b6b94e78cad800f8', 1, 'User used by the systemctl scripts', 'cert');
```

Comment:
with admin role?

Reply:
Yes, admin is needed for now to be able to stop the system. There is as separate Jira that will add a new role with minimal privileges.

Reply:
Fixed in PR-https://github.com/fledge-iot/fledge/pull/1624